### PR TITLE
Edit, document, and review profile tests #1711

### PR DESCRIPTION
**Pull request by willingc**
Addresses #1711 for profile submodule.

Edits to profile tests and depends.py

### CODE REVIEW

File: mysite/profile/tests.py [hunk 1]
```diff
@@ -1565,12 +1566,8 @@ def create_forwarder(address, valid, new_enough_for_display):
             expires_on_future_number = valid and 1 or -1
             stops_being_listed_on_future_number = (new_enough_for_display
                                                    and 1 or -1)
-            expires_on = (datetime.datetime.utcnow() +
-                          expires_on_future_number *
-                          datetime.timedelta(minutes=10))
-            stops_being_listed_on = (datetime.datetime.utcnow() +
-                                     stops_being_listed_on_future_number *
-                                     datetime.timedelta(minutes=10))
```
Comment (ehashman) [inline mysite/profile/tests.py]: I personally prefer this the way it was before, but shrug

File: mysite/profile/tests.py [hunk 2]
```diff
@@ -1891,7 +1879,7 @@ def add_two_people_to_a_project_and_send_emails(
                 email_contexts[participant.pk] = email_context
             output = email_contexts
 
-        return (participants_who_are_news_to_each_other, output)
+        return participants_who_are_news_to_each_other, output
```
Comment (ehashman) [inline mysite/profile/tests.py]: Is this valid without the parens? I've never seen this syntax used before when returning a tuple.

File: mysite/profile/tests.py (hunk 1 above)
Comment (willingc) [inline mysite/profile/tests.py]: Well that's easy. I can revert this change back since now that I'm looking at it more closely, I think you are correct :D

**Comment by ehashman** [comment]
Seeing as tests pass, this is probably safe to merge.

File: mysite/profile/tests.py (hunk 2 above)
Comment (willingc) [inline mysite/profile/tests.py]: Technically, tuples do not require the parens. However for clarity, I have no problem putting them back in.

**Comment by willingc** [comment]
Will open a new PR for the clean up fixes. Thanks @ehashman for the review.

**Pull request merged**